main.rb

```diff
@@ -0,0 +1,88 @@
+require_relative 'planet'
+require_relative 'solar_system'
+
+
+def main
+  solar_system = SolarSystem.new("Sol")
+
+  earth = Planet.new("Earth", "Blue-green", 5.972e24, 1.496e8, "Only planet known to support life")
+
+  mercury = Planet.new("Mercury", "Dark gray", 3.285e102, 35.98, "Smallest and innermost planet.")
+
+  venus = Planet.new("Venus", "Green", 2.759e2, 38.41, "Named after a razor")
+
+  mars = Planet.new("Mars", "Brown", 395e54, 47.34, "Soon a tourist destination")
+
+  jupiter = Planet.new("Jupiter", "Purple", 9472e6, 97.35, "Drops of... crappy song")
+
+  solar_system.add_planet(earth)
+  solar_system.add_planet(mercury)
+  solar_system.add_planet(venus)
+  solar_system.add_planet(mars)
+  solar_system.add_planet(jupiter)
+
+  list = solar_system.list_planets
+  # puts list
+
+#  found_planet = solar_system.find_planet_by_name(planet_choice)
+
+
+
+puts "Welcome to the Solar System Information Station!"
+puts "What would you like to do (enter 1, 2, or 3 ): "
+puts "1. List planets\n2. Add a planet\n3. Exit "
+user_choice = gets.chomp.to_i
+# if user_choice != 1 || user_choice != 2
+# print "Please choose 1 or 2: "
+# user_choice = gets.chomp
+if user_choice == 1
+  list = solar_system.list_planets
+  puts list
+
+  print "Which planet would you like to know about? "
+  planet_choice = gets.chomp
```

Comment:
You need to put this into some sort of loop to prevent the user from typing "1" or something to see a planet.

Keep the user from comiting a typo!